lib/lib-swagger-doc/src/main/java/org/sagebionetworks/swagger/generator/SwaggerSpecJsonGenerator.java

```diff
@@ -0,0 +1,23 @@
+package org.sagebionetworks.swagger.generator;
+
+import org.json.JSONObject;
+import org.sagebionetworks.schema.adapter.JSONObjectAdapterException;
+import org.sagebionetworks.swagger.datamodel.SwaggerSpecModel;
+
+import com.google.gson.Gson;
+
+/**
+ * This generator generates a JSONObject based on a SwaggerSpecModel that conforms to the OpenAPI
+ * specification standards.
+ * 
+ * @author lli
+ *
+ */
+public class SwaggerSpecJsonGenerator {
+	
+		public static JSONObject generateJson(SwaggerSpecModel swaggerSpecModel) throws JSONObjectAdapterException {
```

Comment:
move this to the SwaggerSpecModel 

SwaggerSpecModel .toJson()
new SwaggerSpecModel (String json)